applications/FluidDynamicsApplication/python_scripts/cfd_utility_function_process.py

```diff
@@ -0,0 +1,80 @@
+# Importing the Kratos Library
+import KratosMultiphysics as Kratos
+from .cfd_utility_functions import GetCFDUtilityFunction
+
+
+def Factory(settings, model):
+    if (not isinstance(settings, Kratos.Parameters)):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string"
+        )
+    if (not isinstance(model, Kratos.Model)):
+        raise Exception(
+            "expected input shall be a Model object, encapsulating a json string"
+        )
+
+    return CFDUtilityFunctionProcess(model, settings["Parameters"])
+
+
+class CFDUtilityFunctionProcess(Kratos.Process):
```

Comment:
I'm not sure about exposing these as a process to the user...

Instead, I think we can add the settings to the `turbulence_model_solver_settings` in the `solver_settings`. Then the methods must be automatically called internally.